src/content/api/media-uploads.mdx

```diff
@@ -1,10 +1,9 @@
 ---
 title: Media Uploads
 description: Media Uploads API Reference
-draft: true
 nav:
   path: API
-  order: 7
+  order: 25
```

Comment:
This system seems very cumbersome to insert a document in between another document.
Here are some alternatives that may make it easier.

## Use directories, then prepend files and directories with their order
This is going to reduce the number of documents you need to edit when inserting a document, plus it's far more obvious what documents need to be edited.

## A separate config file which dictates the order
You can have a file that includes an array of file paths in the order you want them displayed. Very easy to change the order, the downside is you need to know how to edit this when adding a new document.

Reply:
Yeah I agree its not great 😅 - I'll add a task to address this

Reply:
It could be nice if each file declared which file it is `after`

Reply:
I agree @IsaacMurtagh I think prepending the files is a nice solution here 💯

Reply:
I'm not sure I understand the directories suggestion - if files/directories are prepended with their order won't this require updating all subsequent file/directory names?

Reply:
@AndyClifford I think what he means is give the directory an index and the page within an index as well. For example `src/content/api/1-assets/1-assets.mdx` and `src/content/api/1-assets/2-asset-transfers.mdx` etc